docs/src/pages/company/software-engineer/software-engineer.md

```diff
@@ -15,7 +15,7 @@ Material-UI started back in 2014 to unify React and Material Design. Today, Mate
 - Type of work: Contractor (long mission, full-time, convertible to an employee position in the future).
 - Start date: Immediately.
 - Location: Remote (preference for UTC-5 to UTC+4).
-- Level: [Senior Engineer or above](https://open.buffer.com/engineering-career-framework/).
+- Level: [Senior Engineer or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit?usp=sharing).
```

Comment:
```suggestion
- Level: [Senior Engineer or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k).
```
Or to avoid a redirect:
```suggestion
- Level: [Senior Engineer or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).
```